cmd/manager/main.go

```diff
@@ -96,7 +100,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	duration := 30 * time.Second
+	duration := defaultSyncInterval
+	if str, ok := os.LookupEnv("LIEUTENANT_SYNC_DURATION"); ok {
+		if d, err := time.ParseDuration(str); err != nil {
+			log.Error(err, "failed to parse LIEUTENANT_SYNC_DURATION")
+		} else if d > 30*time.Second {
+			log.Info("LIEUTENANT_SYNC_DURATION is less than 30s, setting to 30s")
```

Comment:
This should go in another `else` and should say something like:
```suggestion
			log.Infof("LIEUTENANT_SYNC_DURATION is less than %s, setting to %s", defaultSyncInterval, defaultSyncInterval)
```

Reply:
oh right 🙈  mixed up the compare signs...